### PR TITLE
Update docs for Slack and our new release cycle

### PR DESCRIPTION
**Pull request by tas50**
The docs were getting a bit dated

### CODE REVIEW

**Comment by lamont-granquist** [comment]
👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,84 +15,54 @@ We have a 3 step process that utilizes **Github Issues**:
...
-### <a name="cr"></a> Chef Code Review Process
+The Chef Code Review process happens on Github pull requests. See [this article](https://help.github.com/articles/using-pull-requests) if you're not familiar with Github Pull Requests.
 
-The Chef Code Review process happens on Github pull requests. See
-  [this article](https://help.github.com/articles/using-pull-requests) if you're not
-  familiar with Github Pull Requests.
-
-Once you a pull request, the **Chef Engineering Team** or **Chef Core Committers** will review your code
-  and respond to you with any feedback they might have. The process at this point is as follows:
+Once you a pull request, the **Chef Engineering Team** or **Chef Core Committers** will review your code and respond to you with any feedback they might have. The process at this point is as follows:
```
Comment (adragus-inviqa) [inline CONTRIBUTING.md]: Once you *make* a pull request?

Comment (tas50) [inline CONTRIBUTING.md]: Fixed.  Thanks for noticing

**Comment by tas50** [comment]
@lamont-granquist Any reason not to merge this now?

**Pull request merged**